src/jwtf.erl

```diff
@@ -102,10 +102,10 @@ verification_algorithm(Alg) ->
 
 
 validate(Header0, Payload0, Signature, Checks, KS) ->
-    Header1 = props(decode_json(Header0)),
+    Header1 = props(decode_b64_json(Header0)),
```

Comment:
say `b64url` here as `b64` is a slightly different encoding.

Reply:
(and in the commit text, etc)

Reply:
done